### PR TITLE
Managing FTP users in CP

### PR DESCRIPTION
**Pull request by ikidnapmyself**
Add a section to explain how to manage FTP users in CP

### CODE REVIEW

File: docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md [hunk 1]
```diff
@@ -25,7 +25,22 @@ File Transfer Protocol (FTP) and Secure File Transfer Protocol (SFTP) are both w
...
 
-## Managing FTP
+## Managing FTP Users in Control Panel
+
+To manage FTP users in the Control Panel, follow these steps:
+
+- **Navigate to Hypernode:** Go to the Hypernode you want to manage FTP accounts for. In the side menu, locate and click on the "FTP" menu item.
+- **View Existing Users:** If you already have existing users created via the CLI tool, they will be listed on this page. If no FTP users exist, you will see a form to create a new one.
+- **Fill Out the Form:** In the form, you can specify the FTP username, password, and home directory for a new user.
+- **Delete Existing User:** n the user list, you can click the "trash" icon in the row to delete the user.
```
[the remark this answers is not in the document]
Comment (Niutah) [reply docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md:35]: you can type "1." only, not 1 / 2 / 3 / 4 :)

**Review by Niutah — APPROVED** [review]

**Review by hrithikkothari1234 — APPROVED** [review]

File: docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md [hunk 2]
```diff
@@ -25,7 +25,22 @@ File Transfer Protocol (FTP) and Secure File Transfer Protocol (SFTP) are both w
...
+## Managing FTP Users in Control Panel
+
+To manage FTP users in the Control Panel, follow these steps:
+
+1. **Navigate to Hypernode:** Go to the Hypernode you want to manage FTP accounts for. In the side menu, locate and click on the "FTP" menu item.
+1. **View Existing Users:** If you already have existing users created via the CLI tool, they will be listed on this page. If no FTP users exist, you will see a form to create a new one.
+1. **Fill Out the Form:** In the form, you can specify the FTP username, password, and home directory for a new user.
+1. **Delete Existing User:** In the user list, you can click the "trash" icon in the row to delete the user.
+
+**Important Points to Note:**
```
Comment (tdgroot) [inline docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md:37]: More semantic to wrap this block in
````
```{admonition} Important points to note
:class: tip

- If you create a new user with an existing username, the existing user will be overwritten with the new password and home directory.
- Home directories should start with '/' and must exist on your Hypernode.
- The same rules apply to the CLI tool, and these instructions are equivalent.
```
````

[Read more about admonitions here](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html)

Comment (tdgroot) [reply docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md:37]: So you can do things like 
image

**Review by tdgroot — APPROVED** [review]

File: docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md (hunk 2 above)
Comment (ikidnapmyself) [reply docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md]: @tdgroot great, it looks better. can you merge it we are not allowed to merge to the base branch.


    https://docs-ephj21c3q.hypernode.io/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.html


afbeelding

File: docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md [hunk 3]
```diff
@@ -25,9 +25,27 @@ File Transfer Protocol (FTP) and Secure File Transfer Protocol (SFTP) are both w
...
+To manage FTP users in the Control Panel, follow these steps:
+
+1. **Navigate to Hypernode:** Go to the Hypernode you want to manage FTP accounts for. In the side menu, locate and click on the "FTP" menu item.
+1. **View Existing Users:** If you already have existing users created via the CLI tool, they will be listed on this page. If no FTP users exist, you will see a form to create a new one.
+1. **Fill Out the Form:** In the form, you can specify the FTP username, password, and home directory for a new user.
+1. **Delete Existing User:** In the user list, you can click the "trash" icon in the row to delete the user.
+
+```{admonition} Important Points to Note:
+---
+class: tip
```
Comment (tdgroot) [inline docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md:39]:
```suggestion
class: note
```

**Review by tdgroot — APPROVED** [review]

**Pull request merged**